api/v1alpha1/linodemachine_types.go

```diff
@@ -75,6 +81,17 @@ type LinodeMachineSpec struct {
 	CredentialsRef *corev1.SecretReference `json:"credentialsRef,omitempty"`
 }
 
+// InstanceDisk defines a list of disks to use for an instance
+type InstanceDisk struct {
+	// DiskID is the linode assigned ID of the disk
+	DiskID int `json:"diskID,omitempty"`
+	// SizeGB is the size in GB of the disk
+	// +kubebuilder:validation:Required
+	SizeGB int `json:"sizeMB"`
```

Comment:
variable names with GB/MB seems to not match here.

Reply:
good catch, this has now been fixed